framework/ui/internal/uiengine.cpp

```diff
@@ -93,12 +93,6 @@ void UiEngine::setup(QQmlEngine* e)
     importPaths.append(dir.absolutePath());
     m_engine->setImportPathList(importPaths);
 #endif
-#ifdef Q_OS_MAC
```

Comment:
What was wrong with that code?

Reply:
This code rewrites qml import paths and application can't find Qt's modules, therefore, crashed on startup with the message, that QtQuick module not found.

Reply:
ha, quite interesting